### PR TITLE
Fixing InterlaceMode naming conventions

### PR DESCRIPTION
**Pull request by wabiloo**

### CODE REVIEW

File: src/main/java/com/bitmovin/api/encoding/filters/InterlaceFilter.java [hunk 1]
```diff
@@ -16,12 +16,12 @@ public InterlaceFilter()
         this.setType(FilterType.INTERLACE);
     }
 
-    public InterleaveMode getInterleaveMode()
+    public InterlaceMode getInterleaveMode()
```
Comment (jamescyeh) [inline src/main/java/com/bitmovin/api/encoding/filters/InterlaceFilter.java]:
```suggestion
    public InterlaceMode getInterlaceMode()
```

File: src/main/java/com/bitmovin/api/encoding/filters/InterlaceFilter.java [hunk 2]
```diff
@@ -16,12 +16,12 @@ public InterlaceFilter()
...
     {
         return this.mode;
     }
 
-    public void setInterleaveMode(InterleaveMode mode)
+    public void setInterleaveMode(InterlaceMode mode)
```
Comment (jamescyeh) [inline src/main/java/com/bitmovin/api/encoding/filters/InterlaceFilter.java]:
```suggestion
    public void setInterlaceMode(InterlaceMode mode)
```

**Review by jamescyeh — CHANGES_REQUESTED** [review]
two minor changes

**Review by wabiloo — COMMENTED** [review]
Yep, makes sense

**Review by rkersche — APPROVED** [review]

**Pull request merged**